plugins/main/public/controllers/register-agent/services/register-agent-os-commands-services.tsx

```diff
@@ -45,8 +45,13 @@ const getAllOptionalsMacos = (
   return Object.entries(paramNameOrderList).reduce(
     (acc, [key, value]) => {
       if (optionals[value]) {
-        acc += `${optionals[value]}\\n`;
+        acc += `${optionals[value]}`;
       }
+
+      if (Number.parseInt(key) + 1 < Object.entries(optionals).length) {
+        acc += ` && `;
+      }
```

Comment:
**nitpick**: I think this could cause bugs.

EDITED: ![image](https://github.com/wazuh/wazuh-kibana-app/assets/34042064/a271e2a0-b276-474f-8932-19fdf56f6c9e)


**suggestion**: I understand you want to join the declaration of environment variables. I think a cleaner way to do this is using the `Array.prototype.join()` method instead.

Reply:
Fixed in https://github.com/wazuh/wazuh-kibana-app/pull/5888/commits/d5f3d2ca48fe0ae196d97997c769ca5ccfb6ee0d


https://github.com/wazuh/wazuh-kibana-app/assets/6089438/1d941f77-36bb-4aa2-ba7f-3ba4e5fe38a6